app/src/utils/apiHelpers.ts

```diff
@@ -8,8 +8,11 @@ import {
   DEFAULT_PAGE_NUM,
   DEFAULT_SEARCH_TERM,
   COLLECTION_SORT_OPTIONS,
+  DEFAULT_SORT,
+  DEFAULT_FILTERS,
 } from "../config/constants";
 import { fetchApi } from "./fetchApi";
+import { Filter } from "./searchManager";
```

Comment:
import this as a type by `import type { Filter } ...`. This makes it easier to read and know that it's a type and also helps the compiler a bit by giving it a hint of what's imported.